docker/prometheus/ems_alert_rules.yml

```diff
@@ -0,0 +1,41 @@
+# Example Harvest ems alerts
+
+groups:
+  - name: Harvest Ems Alert
+    rules:
+
+      # Alert for Anti-ransomware state change ems
+      - alert: Anti-ransomware state was changed volume
+        expr: last_over_time(ems_events{message="arw.volume.state"} [4w] ) == 1
+        labels:
+          severity: "error"
+        annotations:
+          summary: "Anti-ransomware state was changed to [{{ $labels.op }}] for Volume uuid [{{ $labels.volumeuuid }}]."
```

Comment:
Not a big deal, but wondering if we can do better than copy/paste the same text between summary and description. In the case that we don't have more to say in description would it make sense to not include a description and instead only include a summary?

Reply:
yes, removing description.